src/main/kotlin/org/rust/ide/inspections/RsAssignToImmutableInspection.kt

```diff
@@ -47,4 +50,12 @@ class RsAssignToImmutableInspection : RsLocalInspectionTool() {
     private fun registerProblem(holder: ProblemsHolder, expr: RsExpr, message: String) {
         RsDiagnostic.CannotAssignToImmutable(expr, message).addToHolder(holder)
     }
+
+    private fun registerProblemWithFix(holder: ProblemsHolder, expr: RsExpr, nameExpr: RsExpr?, message: String) {
```

Comment:
Can we merge `registerProblem` and `registerProblemWithFix` into one method where `nameExpr` is null by default?
And maybe `assigneeExpr` in better than `nameExpr`?